careconnect2025/backend/core/src/main/java/com/careconnect/controller/USPSController.java

```diff
@@ -0,0 +1,24 @@
+package com.careconnect.controller;
+
+import com.careconnect.model.USPSDigest;
+import com.careconnect.service.USPSDigestService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/usps")
```

Comment:
this controller should follow the same path route as the others to keep convention. 
`"/v1/api/usps"`